src/qibo/hamiltonians.py

```diff
@@ -186,3 +188,129 @@ def MaxCut(nqubits, dense=True):
     if dense:
         return ham.dense
     return ham
+
+
+from qibo.symbols import X as tspX
+from qibo.symbols import Y as tspY
+from qibo.symbols import Z as tspZ
+class TSP:
```

Comment:
@shangtai thanks for this. Following our discussion, I believe this class should return the hamiltonian object, similarly to the other functions defined in this file. The TSP hamiltonian can then be used inside the tsp model object.

Reply:
hmm... seems that I have a version control problem. I have deleted the TSP code from hamiltonians.py and it is currently only appear in TSP class under model.